content/influxdb/v1.8/introduction/get-started.md

```diff
@@ -1,9 +1,11 @@
 ---
 title: Get started with InfluxDB OSS
 description: Get started with InfluxDB OSS.
+# v2.0 alias below routes old external links here temporarily.
 aliases:
   - /influxdb/v1.8/introduction/getting_started/
   - /influxdb/v1.8/introduction/getting-started/
+  - /influxdb/v2.0/introduction/getting-started/
```

Comment:
For the specific this issue is trying to resolve, we also need to include `getting_started` (with underscore).

```suggestion
  - /influxdb/v2.0/introduction/getting-started/
  - /influxdb/v2.0/introduction/getting_started/
```

Reply:
> For the specific this issue is trying to resolve, we also need to include `getting_started` (with underscore).

Ah, thanks!